_includes/forms-guidance.html

```diff
@@ -17,6 +17,30 @@ <h2 id="general-accessibility-guidance-for-forms">Accessibility guidance</h2>
       ideal, from an accessibility standpoint, because of limited vision that makes it hard to scan from right to left.
     </li>
   </ul>
+  <h2 id="disabled-states-for-components-in-forms">Disabled states for components in forms</h2>
+   <ul class="usa-content-list">
+     <li><strong>Don’t use disabled states, especially for text inputs.</strong> Disabled states have low color contrast, 
```

Comment:
```suggestion
     <li><strong>Avoid disabled states, especially for text inputs.</strong> Disabled states have low color contrast, 
```

Reply:
Let's soften this a little bit, especially since the next bullet outlines some acceptable cases.